test/functions/generic/poi/check-gc-reuse-inits.good

```diff
@@ -6,12 +6,12 @@ check-gc-reuse-inits.chpl:19: warning: User.workIN
 check-gc-reuse-inits.chpl:20: In method 'init=':
 check-gc-reuse-inits.chpl:20: warning: User.workIE
   check-gc-reuse-inits.chpl:42: called as (MyRecord(0)).init=(other: MyRecord(0))
-check-gc-reuse-inits.chpl:22: In function '=':
+check-gc-reuse-inits.chpl:22: In method '=':
 check-gc-reuse-inits.chpl:22: warning: User.workAS
-  check-gc-reuse-inits.chpl:43: called as =(lhs: MyRecord(0), rhs: MyRecord(0))
-check-gc-reuse-inits.chpl:23: In function '<':
+  check-gc-reuse-inits.chpl:43: called as MyRecord.=(lhs: MyRecord(0), rhs: MyRecord(0))
```

Comment:
Ugh, perhaps future work: should say "called as =(....)" (i.e. without "MyRecord.") because one can't call an operator as a method the way this message implies.